crates/re_viewport/src/viewport.rs

```diff
@@ -141,23 +141,35 @@ impl<'a, 'b> Viewport<'a, 'b> {
             &mut blueprint.tree
         };
 
-        let mut tab_viewer = TabViewer {
-            viewport_state: state,
-            ctx,
-            space_views: &mut blueprint.space_views,
-            maximized: &mut blueprint.maximized,
-        };
-
         ui.scope(|ui| {
             ui.spacing_mut().item_spacing.x = re_ui::ReUi::view_padding();
 
             re_tracing::profile_scope!("tree.ui");
-            let tree_before = tree.clone();
+
+            let mut tab_viewer = TabViewer {
+                viewport_state: state,
+                ctx,
+                space_views: &mut blueprint.space_views,
+                maximized: &mut blueprint.maximized,
+                edited: false,
+            };
+
+            // We simplify before we take the before-snapshot, because we
+            // don't want to detect automated simplifcation as user edits.
```

Comment:
there's no before snapshot anymore. At least not in the form of `let tree_before = tree.clone()`. What does this refer to now?

Reply:
it refers to ghost-code